mk_boards.h

```diff
@@ -0,0 +1,43 @@
+#ifndef _MK_BOARDS
+#define _MK_BOARDS
+
+#include <stdint.h>
+#include "mk_com.h"
+
+#define BOARDS_NUM (sizeof(poe_boards)/sizeof(poe_boards[0]))
+#define POE_BOARD_NAME_LEN 32
+#define SPIDEV_MAX_LEN 20
+
+struct poe_board {
+	char name[POE_BOARD_NAME_LEN];
+	int proto_ver;
+	char spidev[SPIDEV_MAX_LEN];
+	int ports_num;
+	int port_state_map[POE_CMD_PORTS_MAX];
+};
+
+/*************************************************************************************
+	все изместные нам PoE устройства */
+const struct poe_board poe_boards[ ] = {
+	{ /* RouterBOARD 750P r2, RouterBOARD 750UP r2*/
+		.name = "rb-750p-pbr2",
+		.proto_ver = 2,
+		.spidev = "/dev/spidev0.2",
+		.ports_num = 4,
+		.port_state_map = {0xd, 0xc, 0xb, 0xa}
+	}, { /* RouterBOARD 960PGS */
+		.name = "mikrotik,routerboard-960pgs",
+		.proto_ver = 3,
+		.spidev = "/dev/spidev0.2",
+		.ports_num = 4,
+		.port_state_map = {0xd, 0xc, 0xb, 0xa}
+	}, { /* RouterBOARD RB5009UPr+S+IN */
+		.name = "mikrotik,rb5009upr",
```

Comment:
Some RB5009 branches call it just `mikrotik,rb5009` - I'm using Arie's branch: https://github.com/Arie/openwrt-rb5009/tree/main